jobs/alertmanager/spec

```diff
@@ -135,7 +135,9 @@ properties:
   alertmanager.route.match:
     description: "A set of equality matchers an alert has to fulfill to match the node"
   alertmanager.route.match_re:
-    description: A set of regex-matchers an alert has to fulfill to match the node
+    description: "A set of regex-matchers an alert has to fulfill to match the node"
+  alertmanager.route.mute_time_intervals:
+    description: "Times when the route should be muted"
```

Comment:
If I see it right that would add the mute_time_interval to the default receiver which is not allowed/ supported according to the documentation (it also failed for me when trying to configure it like that). 
So I think this spec is obsolete as you can only configure the mute_time_intervals for routes defined with property `alertmanager.route.routes`. Which means the mute_time_interval for the specific route would be defined inside there.

Reply:
the example config in the documentation says:

```
# Times when the route should be muted. These must match the name of a
# mute time interval defined in the mute_time_intervals section. 
# Additionally, the root node cannot have any mute times.
# When a route is muted it will not send any notifications, but
# otherwise acts normally (including ending the route-matching process
# if the `continue` option is not set.)
```

Reply:
As you say, I have confirmed that default receiver cannot have mute_time_intervals.
I'll fix it. thanks.